TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    period = 6.03607 #days
</pre><p> </p><p>Wait. Are we just taking this information right from the paper? If so, we'll need to switch to using lightkurve. Typically, we would create a periodogram, then phase fold at the maximum power of the periodogram. (see <a href="https://spacetelescope.github.io/mast_notebooks/notebooks/Kepler/creating_periodograms/creating_periodograms.html?highlight=periodogram" target="_blank">creating periodograms</a> for this workflow)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>okay</p>